### PR TITLE
Add a generic slack api lambda function to avoid needing a custom function for each api call (especially for one-off use cases)

### PR DESCRIPTION
**Pull request by noxasaxon**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [X] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: functions/generic/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+def handle_state(
+    api_method: str,
+    http_verb: str = "POST",
+    params: dict = None,  # type: ignore
+    json: dict = None,  # type: ignore
+    token: str = "",
+):
+    """Create slack channel and invite any provided slack_ids.
```
Comment (ubaniabalogun) [inline functions/generic/lambda_function.py:15]: Doc string here needs updating

File: serverless.yml [hunk 2]
```diff
@@ -86,6 +86,14 @@ functions:
           method: post
           cors: true
 
+  Generic:
+    handler: lambda_function.lambda_handler
+    name: socless_slack_generic
```
Comment (ubaniabalogun) [inline serverless.yml:91]: Recommending `socless_slack_api_request` / `socless_slack_web_api_request` / something else that makes the purpose of this generic a bit clearer

**Review by ubaniabalogun — CHANGES_REQUESTED** [review]

File: functions/generic/lambda_function.py (hunk 1 above)
Comment (noxasaxon) [reply functions/generic/lambda_function.py]: fixed the docstring and added an example of usage

File: serverless.yml (hunk 2 above)
Comment (noxasaxon) [reply serverless.yml]: name has been changed

**Review by ubaniabalogun — APPROVED** [review]

**Pull request merged**